packages/nitro-protocol/contracts/ForceMove.sol

```diff
@@ -85,8 +82,7 @@ contract ForceMove is IForceMove {
             largestTurnNum,
             uint48(now) + fixedPart.challengeDuration,
             challenger,
-            isFinalCount > 0,
-            fixedPart,
+            isFinalCount,
```

Comment:
`isFinalCount > 0` is not sufficient information for a client, who only consumes these events and not `CALLDATA`, to maintain a support for the latest state. They might need that to launch a challenge. This change remedies that and makes it more consistent with emitting _all_ of the `variableParts` as we currently do.